kong/clustering/services/debug.lua

```diff
@@ -0,0 +1,69 @@
+local _M = {}
+
+local get_log_level                = require("resty.kong.log").get_log_level
+local set_log_level                = require("resty.kong.log").set_log_level
+local constants                    = require("kong.constants")
+
+local LOG_LEVELS                   = constants.LOG_LEVELS
+local DYN_LOG_LEVEL_KEY            = constants.DYN_LOG_LEVEL_KEY
+local DYN_LOG_LEVEL_TIMEOUT_AT_KEY = constants.DYN_LOG_LEVEL_TIMEOUT_AT_KEY
+
+
+local function rpc_get_log_level(_node_id)
+  return get_log_level(LOG_LEVELS[kong.configuration.log_level])
+end
+
+
+local function rpc_set_log_level(_node_id, log_level, timeout)
+  local timeout = math.ceil(timeout or 60)
+
+  local cur_log_level = get_log_level(LOG_LEVELS[kong.configuration.log_level])
+
+  if cur_log_level == log_level then
+    local message = "log level is already " .. LOG_LEVELS[log_level]
+    return nil, message
+  end
+
+  local ok, err = pcall(set_log_level, log_level, timeout)
+
```

Comment:
I prefer no new lines, it the line is directly followed with `if` that is related to it.
```suggestion
```

Reply:
Addressed during early refactor.